build-tools/packages/build-cli/src/commands/generate/sourceEntrypoints.ts

```diff
@@ -6,26 +6,20 @@
 import fs from "node:fs/promises";
 import { Flags } from "@oclif/core";
 import JSON5 from "json5";
-import * as resolve from "resolve.exports";
 import type { ExportSpecifierStructure, Node } from "ts-morph";
 import { ModuleKind, Project, ScriptKind } from "ts-morph";
-import type { TsConfigJson } from "type-fest";
 
-import type { Logger, PackageJson } from "@fluidframework/build-tools";
+import type { PackageJson } from "@fluidframework/build-tools";
 
-// AB#8118 tracks removing the barrel files and importing directly from the submodules, including disabling this rule.
-// eslint-disable-next-line import/no-internal-modules
-import { isKnownApiTag } from "../../library/apiTag.js";
-import { ApiLevel, ApiTag, BaseCommand } from "../../library/index.js";
+import { ApiLevel, ApiTag, BaseCommand, isKnownApiLevel } from "../../library/index.js";
 import {
 	readPackageJson,
-	readTsConfig,
 	// AB#8118 tracks removing the barrel files and importing directly from the submodules, including disabling this rule.
 	// eslint-disable-next-line import/no-internal-modules
 } from "../../library/package.js";
 // AB#8118 tracks removing the barrel files and importing directly from the submodules, including disabling this rule.
 // eslint-disable-next-line import/no-internal-modules
-import type { ExportData } from "../../library/packageExports.js";
+import { type ExportData, getExportPathFromPackage } from "../../library/packageExports.js";
```

Comment:
I think we are preferring type block separate from runtime:
```suggestion
import type { ExportData } from "../../library/packageExports.js";
// eslint-disable-next-line import/no-internal-modules
import { getExportPathFromPackage } from "../../library/packageExports.js";
```